src/main/java/org/assertj/core/api/AbstractPathAssert.java

```diff
@@ -0,0 +1,446 @@
+     *     assertThat(dangling).exists(false); // success
+     * </code></pre>
+     *
+     * @param followSymlinks whether symbolic links should be followed
+     *
+     * @return self
+     *
+     * @throws AssertionError see description
+     *
+     * @see Files#exists(Path, LinkOption...)
+     * @see LinkOption#NOFOLLOW_LINKS
+     * @see Files#createSymbolicLink(Path, Path, FileAttribute[])
+     */
+    public S exists(final boolean followSymlinks)
+    {
+        paths.assertExists(info, actual, followSymlinks);
+        return myself;
+    }
+
+    /**
+     * Assert that a given path does not exist
+     *
+     * <p><strong>IMPORTANT NOTE:</strong> this method will NOT follow symbolic
+     * links (provided that the underlying {@link FileSystem} of this path
+     * supports symbolic links at all).</p>
+     *
+     * <p>This means that even if the path to test is a symbolic link whose
+     * target does not exist, this assertion will consider that the path exists
+     * (note that this is unlike the default behavior of {@link #exists()}).</p>
+     *
+     * <p>If you are a Windows user, the above does not apply to you; if you are
+     * a Unix user however, this is important. Consider the following shell
+     * code:</p>
+     *
+     * <pre><code class="sh">
+     *     # Creates a symbolic link "foo" whose nonexistent target is "bar"
+     *     $ ln -s bar foo
+     * </code></pre>
+     *
+     * <p>Given the above, the following assertion  will
+     * <strong>fail</strong>:</p>
+     *
+     * <pre><code class="java">
+     *     // Provided that "foo" is a Path for the above symbolic link
+     *     assertThat(foo).doesNotExist();
+     * </code></pre>
+     *
+     * @return self
+     *
+     * @throws AssertionError path exists
+     *
+     * @see Files#notExists(Path, LinkOption...)
+     * @see LinkOption#NOFOLLOW_LINKS
+     */
+    public S doesNotExist()
+    {
+        paths.assertNotExists(info, actual);
+        return myself;
+    }
+
+    /**
+     * Assert that a given path is absolute
+     *
+     * <p><strong>IMPORTANT NOTE:</strong> on {@link FileSystem}s with
+     * symbolic link support, the fact that a path is absolute <em>does not</em>
+     * guarantee that such a path is canonical. The only guarantee that this
+     * assertion makes is that the given path can be used on this filesystem
+     * to identify an object without having to rely on another path, whether the
+     * target of this path exists or not.</p>
+     *
+     * <p>Also note that the fact that a path is absolute does not mean that it
+     * is {@link Path#normalize() normalized}: {@code /foo/..} is absolute, for
+     * instance, but it is not normalized.</p>
+     *
+     * <p>Sample use:</p>
+     *
+     * <pre><code class="java">
+     *     assertThat(somePath).isAbsolute();
+     * </code></pre>
+     *
+     * <p>Sample paths on Unix and Windows which are absolute, or not:</p>
+     *
+     * <ul>
```

Comment:
I would rather see the various examples in the code section with a comment line before each case (not at the end).